src/components/docs/Buckets/TagBucket/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+import Bucket from '../Bucket';
+
+import s from './styles.module.css';
+
+const TagBucket = ({ icon, tags, title, to }) => (
+  <Bucket className={s.root} to={to}>
+    <img className={s.image} src={useBaseUrl(icon)} />
+    <div className={s.textContainer}>
+      <h4 className={s.title}>{title}</h4>
```

Comment:
1) Why do we need a classname on the h4? 
2) All `h4` tags should be styled the same (taken from a `typography.css` file)
3) Are h3s on the page? We should try to choose a DOM structure that makes sense and avoid choosing a tag because it fits the style we want. h4 might make sense here, not sure.